dll/cpl/usrmgr/lang/fr-FR.rc

```diff
@@ -9,11 +9,11 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL "", IDC_USERS_LIST, "SysListView32", LVS_REPORT | LVS_EDITLABELS | LVS_SINGLESEL |
             LVS_SHOWSELALWAYS | LVS_SORTASCENDING | WS_BORDER | WS_TABSTOP, 7, 7, 238, 85, WS_EX_CLIENTEDGE
-    PUSHBUTTON "A&dd...", IDC_USERS_ADD, 87, 98, 50, 14
-    PUSHBUTTON "&Remove", IDC_USERS_REMOVE, 141, 98, 50, 14
-    PUSHBUTTON "Pr&operties", IDC_USERS_PROPERTIES, 195, 98, 50, 14
-    AUTOCHECKBOX "Require user name and password on system startup.", IDC_USERS_STARTUP_REQUIRE, 10, 118, 200, 14, WS_DISABLED
-    PUSHBUTTON "Reset Password", IDC_USERS_RESET, 165, 200, 80, 14, WS_DISABLED
+    PUSHBUTTON "A&jouter...", IDC_USERS_ADD, 87, 98, 50, 14
+    PUSHBUTTON "&Retirer", IDC_USERS_REMOVE, 141, 98, 50, 14
+    PUSHBUTTON "Pr&opriétés", IDC_USERS_PROPERTIES, 195, 98, 50, 14
+    AUTOCHECKBOX "Nécessite un identifiant et mot de passe au démarrage système.", IDC_USERS_STARTUP_REQUIRE, 10, 118, 200, 14, WS_DISABLED
```

Comment:
```suggestion
    AUTOCHECKBOX "Nécessite un identifiant et mot de passe au démarrage du système.", IDC_USERS_STARTUP_REQUIRE, 10, 118, 200, 14, WS_DISABLED
```

Reply:
(optional, don't apply if the checkbox width cannot be increased without changing the whole layout of the surrounding elements)

Reply:
Yes, checkbox extended accordingly